new/INS1020EFSmissionsmuseum.xml

```diff
@@ -45,9 +45,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPlace>
                 <place type="museum" subtype="institution">
-                    <placeName n="1">Evangeliska Fosterlands-Stiftelsens etnografiska missionsmuseum</placeName>
-                    <placeName n="2">Evangeliska Fosterlandsstiftelsens etnografiska missionsmuseum</placeName>
-                    <country ref="LOC5829Sweden"></country>
+                    <placeName n="1" xml:lang="sv">Evangeliska Fosterlands-Stiftelsens etnografiska missionsmuseum</placeName>
+                    <placeName n="2" xml:lang="sv">Evangeliska Fosterlandsstiftelsens etnografiska missionsmuseum</placeName>
+                    <settlement ref="wd:Q1754"/>
+                    <country ref="wd:Q39"/> 
```

Comment:
I know that the guidelines suggest this possibility but wherever we have a local Bm ID it is better to use it, as then we can better search e.g. if you want to quickly look up for all attestations of Stockholm in the database. At the moment the app is not configured to also look for all attestations of wikidata IDs.

Reply:
Shall I create Stockholm, then? It does not exist yet, as far as I can see :)

Reply:
Thank you, yes, with a link to wd